src/components/PostPageDetail/elements/Content.js

```diff
@@ -250,15 +260,6 @@ const Content = ({
             })}
           </View>
         )}
-        {images_url?.length > 0 && (
-          <View style={styles.containerImage}>
-            <ImageLayouter
-              mode={FastImage.resizeMode.cover}
-              images={images_url || []}
-              onimageclick={onImageClickedByIndex}
-            />
-          </View>
-        )}
         <TopicsChip
           onLayout={handleTopicChipHeight}
           isPdp={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the redundant image rendering logic is a good change. It simplifies the code and prevents unnecessary rendering of images.
```diff
-        {images_url?.length > 0 && (
-          <View style={styles.containerImage}>
-            <ImageLayouter
-              mode={FastImage.resizeMode.cover}
-              images={images_url || []}
-              onimageclick={onImageClickedByIndex}
-            />
-          </View>
-        )}
+          <TopicsChip
+            onLayout={handleTopicChipHeight}
+            isPdp={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->